### ISSUE TITLE
TorchCV

### ISSUE THREAD

**Issue by junxnone** [junxnone/tio#393]
> A PyTorch-Based Framework for Deep Learning in Computer Vision

- **待完善**

- [Image Classification](https://github.com/youansheng/torchcv/tree/master/methods/cls)
    - VGG: Very Deep Convolutional Networks for Large-Scale Image Recognition
    - ResNet: Deep Residual Learning for Image Recognition
    - DenseNet: Densely Connected Convolutional Networks
    - ShuffleNet: An Extremely Efficient Convolutional Neural Network for Mobile Devices
    - ShuffleNet V2: Practical Guidelines for Ecient CNN Architecture Design

- [Semantic Segmentation](https://github.com/youansheng/torchcv/tree/master/methods/seg)
    - DeepLabV3: Rethinking Atrous Convolution for Semantic Image Segmentation
    - PSPNet: Pyramid Scene Parsing Network
    - DenseASPP: DenseASPP for Semantic Segmentation in Street Scenes
    
- [Object Detection](https://github.com/youansheng/torchcv/tree/master/methods/det)
    - SSD: Single Shot MultiBox Detector
    - Faster R-CNN: Towards Real-Time Object Detection with Region Proposal Networks
    - YOLOv3: An Incremental Improvement
    - FPN: Feature Pyramid Networks for Object Detection

- [Pose Estimation](https://github.com/youansheng/torchcv/tree/master/methods/pose)
    - CPM: Convolutional Pose Machines
    - OpenPose: Realtime Multi-Person 2D Pose Estimation using Part Affinity Fields

- [Instance Segmentation](https://github.com/youansheng/torchcv/tree/master/methods/seg)
    - Mask R-CNN

# Reference
- [ ] [Github repo](https://github.com/youansheng/torchcv)
- [ ] [Docs](https://pytorchcv.com/)

**Comment by junxnone** [comment]
#291 #311
